### DISCUSSION TITLE
Put discord community link on the main github page

### DISCUSSION THREAD

**Discussion by johnmcbride** [elsa-workflows/elsa-core#4824] [Ideas]
To help with community help get new users of the project a easier ramp up and provide more discoverability for the project it might be a good idea to provide a link on the main github repository page. Possibly even make channels in the discord server for each package or section to help categorize conversations.

**Comment by sfmskywalker** [comment]
Hey there, thanks for the input!

The home page of the GitHub repo already contains a link to the Discord server, but I guess it might be easy to overlook. Would you lie to submit a PR updating the README file with a proposal for where t put the link?

As for creating a channel per package, concept or section, I like that. Perhaps you have a suggestion for an (initial) list of channel names?
